config.js

```diff
@@ -94,6 +94,11 @@ var config = {
     // input and will suggest another valid device if one is present.
     enableNoAudioDetection: true,
 
+    // Enabling this will show a "Save Logs" link in the GSM popover that can be
+    // used to collect debug information (XMPP IQs, SDP offer/answer cycles)
+    // about the call.
+    // enableSaveLogs: false,
```

Comment:
do we want to have it in the whitelist?

Reply:
Sounds useful but I think it'd defeat the purpose of giving the owner of the deployment the choice of enabling it or not.